taskcluster/scripts/toolkit/compile_qt_6.ps1

```diff
@@ -0,0 +1,71 @@
+$REPO_ROOT_PATH =resolve-path "$PSScriptRoot/../../../"
+$FETCHES_PATH =resolve-path "$REPO_ROOT_PATH/../../fetches"
+
+
+$BIN_PATH = "$REPO_ROOT_PATH/bin"
+
+Set-Location $FETCHES_PATH 
+Invoke-WebRequest -Uri https://download.qt.io/archive/qt/6.2/6.2.3/single/qt-everywhere-src-6.2.3.zip -OutFile qt-everywhere-src-6.2.3.zip
```

Comment:
TODO: I would love to have this be a fetch dep as well ( it would make stuff so much easier, but unzip returns non - 0 when unzipping this on windows)